models/classes/class.TestsService.php

```diff
@@ -75,7 +75,6 @@ class taoTests_models_classes_TestsService
     protected function __construct()
```

Comment:
Constructor should be removed as it serves no purpose anymore

Reply:
In addition, in case of `protected $testClass = null` field is not in use anymore, remove it also.